msdk-io/msdk-io-mzml2/src/main/java/io/github/msdk/io/mzml2/MzMLBinaryDataInfo.java

```diff
@@ -0,0 +1,65 @@
+package io.github.msdk.io.mzml2;
+
+public class MzMLBinaryDataInfo {
```

Comment:
These classes can be package-scope only, because they will be only used by the parser